doc/classes/AcceptDialog.xml

```diff
@@ -21,8 +21,8 @@
 			<argument index="2" name="action" type="String" default="&quot;&quot;">
 			</argument>
 			<description>
-				Add custom button to the dialog and return the created button.
-				The button titled with [i]text[/i] and the [i]action[/i] will be passed to [custom_action] signal when it is pressed.
+				Adds a button with label [i]text[/i] and a custom [i]action[/i] to the dialog and returns the created button. [i]action[/i] will be passed to the [custom_action] signal when pressed.
```

Comment:
Note that if I understood correctly, the doc team has agreed to use `[code]` from now on when mentioning arguments, so please use that instead of `[i]` in future PRs.

Reply:
Thanks, will do.